tests/scripts/preflight-skills-cleanup.sh

```
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+# preflight-skills-cleanup.sh — capture l'état AVANT la série de commits.
+# But : figer la réalité (pas tout faire passer) pour comparer après chaque commit.
+set -uo pipefail
+cd "$(cd "$(dirname "$0")/../.." && pwd)"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)